release/models/lacp/openconfig-lacp.yang

```diff
@@ -310,28 +338,31 @@ grouping aggregation-lacp-members-statistics {
       "Top-level grouping for aggregate members list";
 
     container members {
-      config false;
       description
-        "Enclosing container for the list of members interfaces of
-        the aggregate. This list is considered operational state
-        only so is labeled config false and has no config container";
+        "Configuration and operational state of the aggregate member
+	 interfaces";
 
       list member {
         key "interface";
         description
           "List of member interfaces and their associated status for
-          a LACP-controlled aggregate interface.  Member list is not
-          configurable here -- each interface indicates items
-          its participation in the LAG.";
+          a LACP-controlled aggregate interface";
 
         leaf interface {
           type leafref {
-            path "../state/interface";
+            path "../config/interface";
           }
           description
             "Reference to aggregate member interface";
         }
 
+        container config {
+          description
+            "Operational state data for aggregate members";
+
+          uses aggregation-lacp-members-config;
+        }
+
```

Comment:
Also add `uses aggregation-lacp-members-config;` to the [state container below](https://github.com/openconfig/public/pull/943/files#diff-c9bb1c5ddd819c0196804f71d00fb6245a0eb3996186b677ae2d9f617ef78405R367-R376)